### DISCUSSION TITLE
Global search and email

### DISCUSSION THREAD

**Discussion by awnz** [Intermesh/groupoffice#1061] [Q&A] [answered]
Hi all,

I'm looking at Group-Office as a Zimbra replacement for the homelab/family email server.

Coming from Zimbra, one feature we've really enjoyed is the fast search capability, returning near-instant results from all (even shared) folders.

I see in a screenshot from the [Group-Office documentation](https://groupoffice.readthedocs.io/en/latest/using/search.html), Email is a selectable option for the global search:
![image](https://github.com/Intermesh/groupoffice/assets/8421332/6cd006bd-a963-4829-9322-f6210b3d32d1)

... but when testing, I don't get "E-mail" as something to select to search?
![image](https://github.com/Intermesh/groupoffice/assets/8421332/2891727e-2ccb-471e-a26c-3798d8e48086)

I guess there's an admin setting to enable it, but I've not found it. Should I be able to search emails with this?

I don't think it's relevant, but just in case, I do have `$config['email_allow_body_search'] = true;` set in /etc/groupoffice/config.php.

Thanks,

Andrew

**Comment by mschering** [comment, marked as answer]
Sorry, the global search does not work for e-mail accounts. It only finds emails that have been linked to items inside Group-Office.

**Reply by awnz** [reply]
Thanks - I've found a good workaround in the meantime (as per our other thread at https://github.com/Intermesh/groupoffice/discussions/1050#discussioncomment-7494295) 🙂
